overcooked_ai_py/planning/planners.py

```diff
@@ -1033,13 +1052,9 @@ def get_ml_plan(self, start_state, h_fn, delivery_horizon=4, debug=False):
             cost (int): A* Search cost
         """
         start_state = start_state.deepcopy()
-        if start_state.order_list is None:
-            start_state.order_list = ["any"] * delivery_horizon
-        else:
-            start_state.order_list = start_state.order_list[:delivery_horizon]
 
         expand_fn = lambda state: self.get_successor_states(state)
-        goal_fn = lambda state: state.num_orders_remaining == 0
+        goal_fn = lambda state: state.delivery_rew >= DELIVERY_REW_THRES
```

Comment:
Just to double check, we're currently not using any of this code right?

Reply:
This is right. I am deleting them

Reply:
I deprecated all the parts using the heuristic_fn (and thus the goal_fn)